src/ProviderImplementations/OpenAi/OpenAiModelMetadataDirectory.php

```diff
@@ -53,9 +53,7 @@ protected function parseResponseToModelMetadataList(Response $response): array
         /** @var ModelsResponseData $responseData */
         $responseData = $response->getData();
         if (!isset($responseData['data']) || !$responseData['data']) {
-            throw new RuntimeException(
-                'Unexpected API response: Missing the data key.'
-            );
+            throw ResponseException::fromMissingData('OpenAI', 'data');
```

Comment:
The same change still needs to be made in the `AnthropicModelMetadataDirectory` and in `GoogleModelMetadataDirectory`.